tools/manifest/sourcefile.py

```diff
@@ -410,6 +419,11 @@ def name_is_crashtest(self):
         # type: () -> bool
         return self.type_flag == "crash" or "crashtests" in self.dir_path.split(os.path.sep)
 
+    @property
+    def name_is_print_reftest(self):
+        # type: () -> bool
+        return self.type_flag == "print" or "print" in self.dir_path.split(os.path.sep)
```

Comment:
We didn't spec a directory structure variant in the RFC, did we? Maybe an amendment to make.